backend/app/services/channels/dingtalk/handler.py

```
@@ -271,49 +281,389 @@ async def _process_wegent_message(
+        """
+        self.logger.info(
+            "[DingTalkHandler] Processing command: %s for user %d",
+            command,
+            user.id,
+        )
+
+        if command.command == CommandType.NEW:
+            # Clear conversation task mapping
+            if conversation_id:
+                await self._delete_conversation_task_id(conversation_id)
+            self.reply_text("✅ 已开始新对话，请发送您的消息", incoming_message)
```

Ruff (0.14.14)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)